nncf/common/telemetry_extractors.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+from nncf.telemetry.extractors import CollectedEvent
+from nncf.telemetry.extractors import TelemetryExtractor
+
+
+class ModelProcessedWithStripApi(TelemetryExtractor):
+    def extract(self, _: Any) -> CollectedEvent:
+        return CollectedEvent(name="model_processed", data="strip_api")
```

Comment:
@KodiaqQ, @MaximProshin, @AlexKoff88 I would suggest to introduce `function_call` event to cover calling all NNCF functions as following:
```
return CollectedEvent(name="function_call", data="nncf.strip")
```
I would like to know your opinion.

Reply:
Sounds reasonable to me. 
In this case, we'll have one event with different data values.
The only question is whether it is possible to analyse such an event in the front end.

Reply:
> Sounds reasonable to me. In this case, we'll have one event with different data values. The only question is whether it is possible to analyse such an event in the front end.

@popovaan, could you provide your opinion?

Reply:
I see no problems in usage of common `function_call` event name and it is convenient in terms of making a report, as all values can be collected using only one filter `Event name`==`function_call`.

If you want a separate group of function calls, which are needed to be a separate table in report (for example, function calls that raised error), you can introduce another event name to group them. For example: failed_function_call. Otherwise separate names per method are redundant.

Reply:
@alexsu52 what should we do here then?

Reply:
Updated.